src/api/app/helpers/webui/project_helper.rb

```diff
@@ -1,4 +1,5 @@
 module Webui::ProjectHelper
+  # bento_only, but needed by obs_factory
```

Comment:
Can you move them out to a `Webui::ObsFactoryHelper` or something?

Reply:
Yep. I'll do it.

Reply:
Done. Please re-review.